sandbox/libs/authn/src/main/java/org/opensearch/authn/StringPrincipal.java

```diff
@@ -47,4 +47,11 @@ public int hashCode() {
     public String toString() {
         return "StringPrincipal(" + "name=" + name + ")";
     }
+
+    public boolean equals(StringPrincipal secondStringPrincipal) {
```

Comment:
This class already overrides equals, can this be removed?

Reply:
Yep fixed.